grafana/risingwave-user-dashboard.dashboard.py

```diff
@@ -609,6 +609,10 @@ def section_storage(outer_panels):
 
 def section_streaming(outer_panels):
     panels = outer_panels.sub_panel()
+    mv_filter = "executor_identity=~\".*MaterializeExecutor.*\""
+    table_type_filter = "table_type=~\".*MATERIALIZED_VIEW.*\""
```

Comment:
nits: "table_type=~\"MATERIALIZED_VIEW\"" should be sufficient. No need to do * regex match